en/development/rest.rst

```diff
@@ -91,17 +88,44 @@ controller might look something like this::
         }
     }
 
+In our ``RecipesController``, we have several actions that define the logic
+to create, edit, view and delete recipes. In each of our actions we're using
+the ``serialize`` option to tell CakePHP which view variables should be
+serialized when making API responses. We'll connect our controller to the
+application URLs with :ref:`resource-routes`::
+
+    // in config/routes.php
+    $routes->scope('/', function (RouteBuilder $routes) {
+        $routes->setExtensions(['json']);
+        $routes->resources('Recipes');
+    });
+
+These routes will enable URLs like ``/recipes.json`` to return a JSON encoded
+response. Clients could also make a request to ``/recipes`` with the
+``Content-Type: application/json`` header as well.
+
+Encoding Response Data
+======================
+
+In the above controller, we're defining a ``viewClasses()`` method. This method
```

Comment:
There seems to be a word missing after "This method"

Reply:
oops sloppy plane editing.